templates/blog/blog-card-upcoming.html

```diff
@@ -1,6 +1,6 @@
 {% for article in upcoming %}
   <div class="col-4 blog-p-card--post">
-    <header class="blog-p-card__header--{{article.group.slug}}">
+    <header class="blog-p-card__header blog-p-card__header--{{article.group.slug}}">
```

Comment:
Single classes are much easier to maintain then having to chain them with knowledge of the knock on effect. Could you revert back to extending `%post-card-header` on each modifier please

Reply:
I've reverted that change, but remembered why I made it as I was doing so - in [blog-card.html](https://github.com/canonical-web-and-design/www.ubuntu.com/blob/24b4ff93622b3df8d16e4d6b14b14b02141f6b7a/templates/blog/blog-card.html#L3), that class chain means it falls back to the default border color if we for some reason haven't defined a `blog-p-card__header--` variant based on the group slug. I could look into a better solution for that so we're not chaining there too?

Reply:
I would suggest if an article has no group to have the class `blog-p-card__header` or simple `blog-p-card__header--default` that can contain the default styling